source/mapping-rules/mapping-rule-options/mapping-rule-options.txt

```diff
@@ -95,9 +95,10 @@ mapping rule option.
    :titlesonly:
    :hidden:
 
-   /mapping-rules/mapping-rule-options/new-documents
-   /mapping-rules/mapping-rule-options/embedded-array
-   /mapping-rules/mapping-rule-options/embedded-documents
-   /mapping-rules/mapping-rule-options/mapping-rule-filters
-   /mapping-rules/mapping-rule-options/time-series
-   /mapping-rules/mapping-rule-options/rename-collection
+   New Documents </mapping-rules/mapping-rule-options/new-documents>
+   Embedded Array </mapping-rules/mapping-rule-options/embedded-array>
+   Embedded Documents </mapping-rules/mapping-rule-options/embedded-documents>
+   Mapping Rule Filters </mapping-rules/mapping-rule-options/mapping-rule-filters>
+   Rename a Collection </mapping-rules/mapping-rule-options/rename-collection>
```

Comment:
[Suggestion]: Rename Collection

Reply:
And for next line: Configure Time Series Collection.

Reply:
Done